CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+4.67.0
+==========
+### New Features
+- MSAL.NET now supports acquiring token for Client Credentials flow with a Subject Name Issuer (SNI) certificate over Mutual TLS (mTLS) for first-party applications. See [Issue #4986](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4986)
+- General availability of the new `WithSignedHttpRequestProofOfPossession` API for Signed HTTP Requests, enhancing security and compliance in token handling. See [Issue #4986](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4986)
+- Bumped NET SDK from net6 to net8, aligning MSAL.NET with the latest .NET standards and features. See [Feature Request #5025](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/5025)
+- Removed TokenType from authentication result metadata to streamline token handling processes. See [Issue #4996](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/4996)
```

Comment:
No need to mention this, since TokenType was never part of AuthenticationResultMetadata in an oficial released version.